### DISCUSSION TITLE
CLA mentions feedback contributions

### DISCUSSION THREAD

**Discussion by Rapptz** [zed-industries/zed#6769] [Feature Requests]
Hey, congrats on the open source release. I was quite pleasantly surprised to see it. 

I'm unsure where else to put this but I was wondering what the feedback wording was about in the CLA? It seems a bit odd for it to be there and it wasn't there in any other CLA I checked. The wording, at first glance, made it seem like you need to accept the CLA to provide feedback or open issues but I don't think that's the intent which made me curious what the wording was for in the first place.

For posterity, the clause I'm talking about is this one:

> "Contribution" shall mean Feedback (as defined below), any original work of authorship, and any modifications or additions to an existing work, that is intentionally submitted by You to Company for inclusion in, or documentation of, any of the products owned or managed by Company (the "Work"). For the purposes of this definition, "submitted" means any form of electronic, verbal, or written communication sent to Company or its representatives, including but not limited to communication on electronic mailing lists, source code control systems, and issue tracking or collaboration systems that are managed by, or on behalf of, Company for the purpose of discussing and improving the Work, but excluding communication that is conspicuously marked or otherwise designated in writing by You as "Not a Contribution."
> “Feedback” means suggestions, comments, improvements, software code / snippets or other information submitted to, shared with or otherwise made available to Zed or its contributors with respect to, or in connection with the use or interaction with, the Work, Zed Network Based Service or Editor technology (as defined by Zed and as further described within Zed’s End User License Agreement located at https://zed.dev/eula).

I asked this in the Discord server originally and got a response from someone saying that it was a leftover clause from the private fireside hack chats. However, I didn't get any official response to this so I figured I'd open an issue to bring it to attention. If this is leftover wording then it should be possible to remove it.

**Comment by JosephTLyons** [comment]
Hey @Rapptz, our head of operations weighed in on the lingo here and helped to demistify it for us. What we have is a CLFA, or a Contributor License and Feedback Agreement. With respect to feedback, the CLFA was written to allow us to share feedback provided to us within Zed. For example, if we host a Fireside Hack event, and we capture the event through a screenshare, we have permission to post a YouTube video that includes text from users who are chatting and audio of users who might be talking during the event. I dont think feedback in general, through GitHub Issues or anything outside of Zed, is something we are concerned with, I think the CLFA is really just about making sure we are covered if we want to share content that might include users who are in channels with us.

**Reply by Rapptz** [reply]
I think it makes sense to have this be explicit in the CLA since the wording right now feels like it's casting a wider net than intended.

**Reply by mikayla-maki** [reply]
I disagree that this is wider than normal. Take Microsoft's CLA: https://opensource.microsoft.com/pdf/microsoft-contribution-license-agreement.pdf, which uses similar wording under 'Submit', or the Google CLA: https://cla.developers.google.com/about/google-individual, which uses similar wording under 'Contribution'.

The way we've constructed this is standard across the industry, and we are enforcing it where it matters. Consider this the final answer.

**Reply by Rapptz** [reply]
The last sentence felt rather harsh for no reason, but I can "accept" your answer as-is without the harsh tone.

That aside, both those CLAs if you Ctrl + F "Feedback" you get no results, which is what I was discussing in this post. I was not discussing the submission or general contribution definitions.
